api/src/main/java/org/grammatek/apis/UsersApi.kt

```diff
@@ -35,8 +40,9 @@ import org.grammatek.infrastructure.RequestConfig
 import org.grammatek.infrastructure.RequestMethod
 import org.grammatek.infrastructure.ResponseType
 import org.grammatek.infrastructure.Success
+import org.grammatek.infrastructure.toMultiValue
 
-class DevelopersApi(basePath: kotlin.String = org.grammatek.apis.DevelopersApi.Companion.defaultBasePath) : ApiClient(basePath) {
+class UsersApi(basePath: kotlin.String = defaultBasePath) : ApiClient(basePath) {
```

Comment:
although this is a generated file, I don't understand the class name: UsersApi ? Where is the name derived from ? Could we name it better, e.g. CorrectApi or SpellingApi or similar ?

Reply:
Not really derived from anything, it's a name convention for API available to users in the same sense that DevelopersApi. Both fit in my opinion but the naming here can be anything really.

Reply:
So if it can be anything, I'd vote for CorrectApi.